test/ConfigManagerTest.cpp

```diff
@@ -78,12 +50,12 @@ TEST(ConfigManagerTest, CreateConfigurationOptionExceptionTest) {
 
   // Configuration options for testing.
   int notUsed;
-  config.createConfigOption<int>(
-      {"Shared_part", "Unique_part_1", "Sense_of_existence"}, "", &notUsed, 42);
+  config.addOption({"Shared_part", "Unique_part_1", "Sense_of_existence"}, "",
```

Comment:
Use the `"something"s` syntax consistently for less confusion.

Reply:
Done, see [86ee1f0](https://github.com/ad-freiburg/qlever/pull/1040/commits/86ee1f0392b0b90f7e02489d959182bf5807fbff).